deoxys/src/modes.rs

```diff
@@ -299,10 +321,10 @@ where
                 tweak[8..].copy_from_slice(&(index as u64).to_be_bytes());
 
                 if data.len() == 16 {
-                    let mut block = [0u8; 16];
+                    let mut block = Block::default();
```

Comment:
For consistency I think it's worth to change this method to use `chunks_exact` and split whole blocks and tail processing similarly to `encrypt_decrypt_message`.